partials/_v2_transition_banner.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="border border-violet-600 bg-violet-50 rounded-l p-4 my-4 text-base text-navy">
+We're starting to roll out our new Apps V2 platform to new organizations. Where there's any difference in instructions between Apps V2 and the original Nomad-orchestrated app platform, we'll supply the Nomad-specific variant either inline or at the end of the doc.
+<br><br>
+You can check if your org defaults to Apps V2 deployments with `fly orgs apps-v2 <org-slug>`. Get your organization slug using `fly orgs list`.
```

Comment:
Hi @catflydotio ! Can you check the command `fly orgs apps-v2 <org-slug>`? I'm at the latest flyctl version, and I'm only getting a proper response when I use ``fly orgs apps-v2 show <org-slug>` instead

Reply:
Absolutely a typo, thank you!